src/App.js

```diff
@@ -20,9 +20,27 @@ function App() {
   }
 
   const sortAlphabetically = () => {
-    const newList = contacts.sort((x, y) => {
-      
-    })
+
+    const newList = [...contacts.sort((x, y) => 
+     x.name.localeCompare(y.name))]
+
+     setContacts(newList)
+  }
```

Comment:
You should firstly clone the array and then sort it , rather than cloning whole expression

```
    const newList = [...contacts].sort((x, y) => 
     x.name.localeCompare(y.name))
```